### DISCUSSION TITLE
Single plane computation

### DISCUSSION THREAD

**Discussion by Nicola-Fonzi** [su2code/SU2#1144] [Q&A] [answered]
Dear all,
I am working with the Benchmark Supercritical Wing test case, from the 2nd aeroelastic prediction workshop. The model is a rectangular wing, with symmetry plane in the middle, working at transonic conditions.

For this test case, different meshes are available from [here](https://nescacademy.nasa.gov/workshops/AePW2/public/BSCW/analystsInfo/grids#grids). I am using _Coarse Mixed node based_ but I also tried with the medium one. 

The computations seem to run smoothly and the results (I tried to compare steady state results) match with experimental data. However, I noticed that in the log file the check for a single plane with euler/symmetry boundary conditions always states that there is NOT a single plane.

I also tried to do the meshes again from scratch, to be sure that the plane in the middle was indeed a plane. Still the same issue in the log file...

May I ask you if you have ideas about the source of the issue?

I am not sure if the issue is related to compiled program, but in case:
OS: CentOS 7
SU2 version: 7.0.8
gcc version: 10.2.0
openMPI: 4.0.4

Thank you in advance

Nicola Fonzi

**Comment by Nicola-Fonzi** [comment]
Quick update:
I was now performing mesh deformation (new solver) with the mentioned mesh, using symmetric boundary conditions, and the solver did not complain (should raise error if the symmetry plane is not actually a plane). Is it possible there is a small error in the boundary conditions check?
The only thing that I noticed is that there is a warning about the real residual norm differing from the calculated residual norm after deformation, but I am not sure if this is somehow related.

**Comment by pcarruscag** [comment, marked as answer]
It could be that the tolerance for checking the flatness of a boundary is too tight.
In any case the actual implementation of the boundary is the same.
The deformation and FE solvers will only complain if the symmetry is not axis-aligned.

**Comment by Nicola-Fonzi** [comment]
You are right! I will check the tolerance, but as you said the complaining is related to the axis-aligned constraint.
Thank you!
